### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
This is a microverse project where I had to add a menu to the mobile version of the Portfolio web page:
*Add javascript lint files.
*Add a script.js file.
*Update .HTML and .css file.
*Add Disable.svg image file.
*When the user clicks (or taps) the hamburger button, the mobile menu appears.
*When the user clicks (or taps) the close (X) button, the mobile menu disappears.
*When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
*When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,32 +25,33 @@
...
+                </button>
             </nav>
         </header>
 
         <section id="s-1" class="flex">
             <ul class="flex navlinks">
-                <li>
-                    Portfolio
+                <li class="navlink-items">
+                    <a href="#Works-grid">Portfolio</a>
```
Comment (DanteAlonsoHT) [inline index.html:41]: In your desktop version, after I click on any nav link, the whole menu disappears, likely your functionality in your mobile version made this new bug in your website, kindly fix it 😉 

- [ ] Your nav bar should be there even if you click any link before.

**Before clicking any navbar link ( in your desktop version)**

image

**After clicking any navbar link ( in your desktop version)**

image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+menuBtn.addEventListener('click', () => {
+  menuBtn.classList.toggle('active');
+  menuList.classList.toggle('active');
+
+  if (menuBtn.classList.contains('active')) {
+    menuIcon.src = './images/Disabled.svg';
+  } else {
+    // eslint-disable-next-line no-restricted-globals
+    location.reload();
```
Comment (DanteAlonsoHT) [inline script.js]: - [ ] [OPTIONAL] If you can find a way to avoid this eslint-disable comment, it will look be better 😉

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @hamid-murambiwa and @KDlamini, 

Amazing so far! I liked your work! 💯 
There is only one change required, you are almost there! 😄 

## Required Change ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please, all your .svg, .png, .jpeg, .gif should be inside your "img" or "image" folder to have more order in your images used in your repository.
- For your .css you can create "css" folder to save all your .css files used in your repository.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html:41]: * done 👍

**Comment by Davidosky007** [comment]
## Hi @hamid-murambiwa @KDlamini @basnetlaxmi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
thank-you-cr-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**